ui_framework/doc_site/src/views/form_gallery/form_gallery.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Does this example need to be a sandbox? Can we just make this a regular page instead? That way the code examples will be easier to navigate.

Reply:
Can I do this as part of the build the doc layer in kui PR? it's kind of related.

Reply:
TBH I think it makes more sense to do it in this PR. Is there any reason to block it on the changes to the doc layer?

Reply:
Mostly layout. It's just a matter of me having to move it over, then likely do some reformatting again once it's in. The current layout classes a lot when I do mixed component stuff. Similar problem to the tyography thing. I'd rather make the doc layer more flexible with layouts so I can make the layout more custom when I need it.

Reply:
OK, sounds good.